ui/app/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -52,22 +52,43 @@ export default class PermissionPageContainerContent extends PureComponent {
           { redirect ? null : <h1>{domainMetadata.name}</h1> }
           { redirect ? null : <h2>{requestMetadata.origin}</h2> }
         </section>
-        { permissionRejected
-          ? <span className="permission-approval-visual__reject" ><i className="fa fa-times-circle" /></span>
-          : <span className="permission-approval-visual__check" />
+        { redirect
+          ? permissionRejected
+            ? <span className="permission-approval-visual__reject" ><i className="fa fa-times-circle" /></span>
+            : <span className="permission-approval-visual__check" />
+          : null
+        }
+        { redirect
+          ? <img className="permission-approval-visual__broken-line" src="/images/broken-line.svg" />
+          : null
+        }
+        { redirect
+          ? (
+            <section>
+              <div className="permission-approval-visual__identicon-container">
+                <div className="permission-approval-visual__identicon-border" />
+                <Identicon
```

Comment:
This is no longer present in the designs. This page should only show the site icon. Everywhere where `selectedIdentities[0]` is referenced should be removed - the first account in the set isn't highlighted anywhere in the designs.

Reply:
Went away in 14c717f4f

Reply:
Well, some of the `selectedIdentities` references remain, but I will remove those that are unnecessary in a future PR where I do all final updates to designs

Reply:
All unnecessary references to `selectedIdentities[0]` have now been removed